persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -66,6 +79,10 @@ class ConversationDAOImpl(
         )
     }
 
+    override suspend fun updateConversationGroupState(groupState: ConversationEntity.GroupState, groupId: String) {
```

Comment:
is it safe and enough that we update the groupState only based on the groupId? or is it better to add the conversationId as well?

Reply:
It's safe since we only need deal with the `groupState` for MLS conversations which always have a groupId. But it doesn't really matter I can switch to conversationId as well.

Reply:
That would be enough if we are assured of it.